client/src/graphs/templates/ui/TemplateItem.vue

```diff
@@ -0,0 +1,54 @@
+<script setup lang="ts">
+import GButton from "@ui/graph/button/GButton.vue";
+import type { GraphTemplate } from "../types";
+import { ref, toRefs } from "vue";
+import colors from "@utils/colors";
+
+const props = defineProps<{
+  template: GraphTemplate;
+  loadTemplate: (id: string) => void;
```

Comment:
id call these load and delete if we already know the context its in. however delete is a keyword conflict so maybe remove?